### DISCUSSION TITLE
Welcome to ferris-elf Discussions!

### DISCUSSION THREAD

**Discussion by proegssilb** [proegssilb/ferris-elf#72] [Announcements]
## 👋 Welcome!

  I've enabled Discussions so that we have a place other than Discord to talk about things and iron out details.

  Some potential bits of content that might go here rather than in an Issue or Pull Request:
  * Ask questions you’re wondering about.
  * Share ideas that aren't quite filled-in enough for an issue.
  * Engage with other community members.

If people feel like introducing themselves, they can do so right here in this discussion.

**Comment by joaomarcosjova** [comment]
##Hello, I'm Marcos
